ERCS/erc-7715.md

```diff
@@ -0,0 +1,466 @@
+---
+eip: 7715
+title: Request Permissions from Wallets
+description: Adds JSON-RPC method for requesting permissions from a wallet
+author: Luka Isailovic (@lukaisailovic), Derek Rein (@arein), Dan Finlay (@danfinlay), Derek Chiang (@derekchiang), Fil Makarov (@filmakarov), Pedro Gomes (@pedrouid)
+discussions-to: https://ethereum-magicians.org/t/erc-7715-request-permissions-from-wallets/20100
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-05-24
+requires: 4337, 5792, 7679, 7710
+---
+
+## Abstract
+
+We define a new JSON-RPC method `wallet_issuePermissions` for DApp to request permissions from a wallet in order to execute transactions on users’s behalf.  This enables two use cases:
+
+- Executing transactions for users without a wallet connection.
```

Comment:
How would scheduled/recurring transactions work with this ERC? Is that another permission type, or can it be used with the existing ones listed?

Reply:
I assume the user MUST issue permissions with a long expiry time or undefined for permissions that don't expire.

Reply:
Makes sense!

Reply:
Any actions here or should we close this thread?